src/Core/Cache/Factory/Cache.php

```diff
@@ -92,7 +69,7 @@ public function __construct(BaseURL $baseURL, IManageConfigValues $config, Datab
 	 */
 	public function createDistributed(string $type = null): ICanCache
 	{
-		if ($type === Enum\Type::APCU) {
+		if ($type === Type\APCuCache::$NAME) {
```

Comment:
If the type check is warranted, why not check the class name instead?

Reply:
Because the class name isn't used as strategy name. It's the key of the mapping table of `strategies.config.php`